permissions/plugin-kubernetes-credentials.yml

```diff
@@ -9,4 +9,4 @@ developers:
 - "sirstrahd"
```

Comment:
Still need approval here.

Reply:
Do you need the approval from all of us or is mine enough ? Else I can contact him

Reply:
Just need one

Reply:
@maxlaverse do you approve this?

Reply:
Yes, I do approve this update of security contact